datastream-server/src/test/java/com/linkedin/datastream/server/assignment/TestStickyMulticastStrategy.java

```diff
@@ -475,6 +475,31 @@ public void testTriggerRebalanceWhenDeletingDatastream() {
         newAssignment.get(instancesBySize.get(instances.size() - 1)).size());
   }
 
+  @Test
+  public void testExtraTasksAreNotAssignedDuringReassignment() {
+    String[] instances = new String[]{"instance1"};
```

Comment:
Nit: spacing missing from line.

Reply:
sorry, not able to figure out where is space missing?

Reply:
After the opening curly brace and before the closing curly brace.